test/unit/cookie.test.js

```diff
@@ -194,4 +194,135 @@ describe('Cookie', function () {
             })).valueOf()).to.eql('this is a cookie value');
         });
     });
+
+    describe('valueOf', function () {
+        it('should handle malformed URI sequence', function () {
+            expect((new Cookie({
+                name: 'blah',
+                value: '%E0%A4%A'
+            })).valueOf()).to.eql('%E0%A4%A');
+        });
+    });
+
+    describe('unparseSingle', function () {
+        it('should return only the value in case of no cookie name specified', function () {
+            var rawCookie = {
+                    expires: 1502442248,
+                    hostOnly: false,
+                    httpOnly: false,
+                    key: 'foo',
```

Comment:
We have specified a `key` here. Shouldn't it be used for `name`? 🤔

Reply:
I think this might be a bug? We allow using `key` in place of `name` when creating a cookie. Ref: [/lib/collection/cookie.js#L125](https://github.com/postmanlabs/postman-collection/blob/16cc6f821a2af99d979477f9d97432b1d5362090/lib/collection/cookie.js#L125).

@codenirvana views?

Reply:
Yes, this looks like a mistake but it's fine for now. I will take a look at this.